src/audio/copier/copier.c

```diff
@@ -675,7 +675,7 @@ static int copier_comp_trigger(struct comp_dev *dev, int cmd)
 	dai_copier = pipeline_get_dai_comp_latency(dev->pipeline->pipeline_id, &latency);
 	if (!dai_copier) {
 		comp_err(dev, "failed to find dai comp or sink pipeline not running.");
-		return ret;
+		return -ENODEV;
```

Comment:
I don't fully understand the function, but based on test results, a more appropriate changed would be to downgrade "comp_err" to comp_warn or even lower as this does not seem to be an error. Seems this condition happens regularly during use-cases. Ret must be zero here, so existing code returns success here.

Reply:
Ok, should I change this to return 0 then, and downgrade comp_err() to comp_warn() ? I thought this would be an error since there is a lot of code still left in the function, but executing it requires dai_copier being found.